latex-zh.css

```
@@ -0,0 +1,71 @@
+    font-weight: 700;
+    font-display: swap;
+}
+
+:root {
+    --font-family:
+        "Times New Roman", "TeX Gyre Termes", "Liberation Serif",
+        "Nimbus Roman", "Songti SC", "STSong",
+        "SimSun", "NSimSun", "Source Han Serif SC", "Noto Serif CJK SC",
+        serif;
+    --sans-font:
+        Arial, "Helvetica Neue", Helvetica, "Liberation Sans", "Nimbus Sans",
```

Stylelint (17.14.0)

[error] Expected "Arial" to be "arial" (value-keyword-case)

(value-keyword-case)